src/components/pagination/pagination.tsx

```diff
@@ -183,6 +212,7 @@ export class Pagination {
   }
 
   renderPage(start: number): VNode {
+    const lang = this.globalAttributes["lang"] || document.documentElement.lang || "en";
```

Comment:
I could see the fallback options for the document and "en" being used a lot. I wonder if we can DRY this up so that every component getting "lang" doesn't have to set these fallbacks. What do you think @jcfranco?